src/components/Layout/Layout.scss

```diff
@@ -661,10 +661,26 @@ header {
         font-size: 1.25em;
         line-height: 1.75em;
         opacity: 0.9;
+        li {
+            font-size: 1em;
+        }
     }
     li {
         margin-bottom: calc(1.45rem / 2);
     }
+    img.large {
+        max-width: 150%;
+        margin-left: -25%;
+    }
+}
+
+@media screen and (max-width: 800px) {
+    .blog-post-content {
+        img.large {
+            max-width: 100%;
+            margin-left: 0;
+        }
+    }
 }
```

Comment:
Adding support for larger inline images in blog posts at larger resolutions.
![Screen Shot 2021-09-27 at 18 46 52](https://user-images.githubusercontent.com/328367/134959762-e99ad2dd-4ace-4aea-9d21-351f3418a13a.png)
![Screen Shot 2021-09-27 at 18 47 02](https://user-images.githubusercontent.com/328367/134959768-4807f491-4b90-47bf-9f3d-e065b44877ee.png)

Reply:
@smallbrownbike - you okay with this?